### PR TITLE
Add Series.diff()

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR adds diff in Series.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
Tests passed

**Comment by RainFung** [comment]
Thanks i close it

**Comment by HyukjinKwon** [comment]
Nope, this adds into Series. Your PR added into DataFrame only.

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -2318,6 +2318,83 @@ def describe(self, percentiles: Optional[List[float]] = None) -> 'Series':
...
+
+        Returns
+        -------
+        diffed : DataFrame
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'a': [1, 2, 3, 4, 5, 6],
+        ...                    'b': [1, 1, 2, 3, 5, 8],
+        ...                    'c': [1, 4, 9, 16, 25, 36]})
```
Comment (ueshin) [inline databricks/koalas/series.py]: need `columns=[...]` if we print `df`?
Also could you add it for the equivalent example for `DataFrame`?

**Comment by HyukjinKwon** [comment]
Thanks!

**Pull request merged**